src/components/TodoList/TodoList.tsx

```diff
@@ -16,85 +28,51 @@ export const TodoList: React.FC = () => (
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+      {todos.map(todo => (
+        <tr
+          key={todo.id}
+          data-cy="todo"
+          className={classNames(
+            { 'has-background-info-light': selectedTodo === todo.id },
+          )}
+        >
+          <td className="is-vcentered">{todo.id}</td>
+          <td className="is-vcentered">
+            {todo.completed
+              && (
+                <span className="icon" data-cy="iconCompleted">
+                  <i className="fas fa-check" />
+                </span>
+              )}
+          </td>
+          <td className="is-vcentered is-expanded">
+            <p className={classNames(
+              { 'has-text-success': todo.completed },
+              { 'has-text-danger': todo.completed },
+            )}
+            >
+              {todo.title}
+            </p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button
+              data-cy="selectButton"
+              className="button"
+              type="button"
+              onClick={() => onSelect(todo.id)}
+            >
+              <span className="icon">
+                <i className={classNames(
+                  'far',
+                  { 'fa-eye': selectedTodo !== todo.id },
+                  { 'fa-eye-slash': selectedTodo === todo.id },
+                )}
+                />
```

Comment:
to my preference ternary is more concise
```suggestion
                <i className={classNames(
                  'far',
                  selectedTodo === todo.id
                    ? 'fa-eye-slash'
                    : 'fa-eye'
                )}
                />
```